rs/sns/init/src/lib.rs

```diff
@@ -3432,36 +3401,40 @@ mod test {
     }
 
     #[test]
-    fn test_number_of_sns_neuorns_for_direct_participants_cannot_exceed_limit() {
+    fn test_validate_participation_constraints() {
```

Comment:
Does this test now test something which previously was not tested?
I'm confused because in the code under test, a requirement is just removed.
So I would expect the test that tests that requirement to also be removed.
So I'm not sure what this test is testing now.

Reply:
This is the only unit test for `validate_participation_constraints`, so I'd rather keep it.